bin/jsonnet-lint

```diff
@@ -7,4 +7,5 @@ DOCKER_USER="$(id -u):$(id -g)"
 # the dashboard directory is mounted in the container working directory, e.g.
 # src/dashboards is mounted in /app/dashboards (with /app as the working
 # directory).
-DOCKER_USER=${DOCKER_USER} docker-compose run --rm app jsonnet-lint "${@//src\//}"
+COMMAND="for file_name in ${@//src\//}; do jsonnet-lint \$file_name ; done"
+DOCKER_USER=${DOCKER_USER} docker-compose run --rm app /bin/sh -c "$COMMAND"
```

Comment:
I was hesitating about this, calling this script multiple times makes multiple `docker-compose run` calls which is a bit slower than one call with a loop inside. WDYT ?

Reply:
You can passe multiple file paths as arguments and they will all be treated at once:

```
$ bin/jsonnet-lint file1.jsonnet file2.jsonnet
```

Reply:
This would be really cool! Thanks) Actually, `jsonnet-lint` just merged 4 days ago the support for multiple files [in this commit](https://github.com/google/go-jsonnet/commit/51daeb3229e488ce4a57d16bcfd77b6de896d03c))
But unfortunately they don't have a tagged version for now. Do we update our `Dockerfile` to follow the master branch ? `go get github.com/google/go-jsonnet/cmd/jsonnet-lint@master` ?

Reply:
Apropos, the latest version of jsonnet-lint works with multiple files BUT only when they are independent :disappointed:
As soon as one file imports another (and both are present in the list to lint) the linter exits with an error.
There is an open issue regarding the raised error [here](https://github.com/google/go-jsonnet/issues/544).